doc/sphinx/source/add-ons.rst

```diff
@@ -118,36 +120,36 @@ A UI theme. See :doc:`themes` for details.
 Restrictions
 ------------
 
-The order of add-ons matters. Add-ons can be reordered in the in-game add-ons manager. Enabled add-ons will be executed from top to bottom. If, for example, you enable one add-on A that adds a new worker type that requires experience and another add-on B that modifies all workers’ experience thresholds, the new worker’s experience will be modified by B if and only if B is loaded later than A.
+The order of add-ons matters. Add-ons can be reordered in the in-game add-ons manager. Enabled add-ons are executed from top to bottom. If, for example, you enable one add-on A that adds a new worker type that requires experience and another add-on B that modifies all workers’ experience thresholds, the new worker’s experience is modified by B if and only if B is loaded later than A.
 
```

Comment:
> ... the new worker’s experience is modified by B if and only if B is loaded later than A.
```suggestion
... the new worker’s experience is modified by B because B is loaded after A.
```